src/common/components/atoms/progress.tsx

```diff
@@ -4,31 +4,33 @@ import * as ProgressPrimitive from "@radix-ui/react-progress";
 import { mergeClasses } from "@/common/lib/utils/mergeClasses";
 
 type CustomProgressProps = {
-  indicatorClassName?: string;
+  indicatorStyles?: object;
```

Comment:
It would be nice if we could know all of the possible keys, but that isn't worth doing now. That said, these should all be values that are strings or numbers, rights? Could we do `Record<string, string | number>`?

Reply:
Updated 👍